Makefile

```diff
@@ -324,12 +322,6 @@ envtest: $(ENVTEST) ## Download setup-envtest locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	GOBIN=$(CACHE_BIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
-.PHONY: envsubst
```

Comment:
We should keep envsubst, we're using that in the Tiltfile: https://github.com/linode/cluster-api-provider-linode/blob/main/Tiltfile#L16

Reply:
@AshleyDumaine  if we only need envsubst for the Tiltfile we should absolutely rip it out fo there as well. is there anywhere else we are using envsubst explicitly?